.github/workflows/ci.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       run: .github/workflows/install_deps.sh amici pysb
 
     - name: Run tests
-      timeout-minutes: 25
+      timeout-minutes: 35
```

Comment:
if it takes that long out of a sudden, something should be wrong. was 15 not long ago

Reply:
The number of test cases increased:
* additional test cases 
* running for both SBML and PySB models